justfile

```diff
@@ -10,10 +10,10 @@ default:
 ###########
 
 # Sleep for given time showing the given message as long as given condition is met
-@_loop condition message time="5":
-    while [ {{ condition }} ]; do \
-        echo "{{ message }}" && sleep {{ time }}; \
-    done
+@_loop condition message timeout="5m" time="5":
```

Comment:
5 minutes is still a long time! Is that a reasonable amount of time to expect ES to take to boot up? I would have thought like 2 minutes max but maybe it could take a lot longer on smaller CPUs (or even in the action runners).